Validation: Duplicate alt text and missing description error fixes. (#1440)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/advanced/restricting-the-drawing-surface-in-gdi.md

---
title: "Restricting the Drawing Surface in GDI+"

> description: Learn how to restrict the drawing surface in GDI+ using regions, which can be constructed from paths and contain the outlines of strings.

Reviewer: ```suggestion
description: Learn how to restrict the drawing surface in GDI+ using clipping regions in Windows Forms.
```
